src/main/java/SearchEngine.java

```diff
@@ -46,24 +46,18 @@ private void printMenu() {
                 0. Exit""");
     }
 
-    private void findPeoples(Map<String, List<Integer>> linesIndexesByWord) {
+    private void findPeoples(Map<String, List<Integer>> linesIndexesByWord) throws ArrayIndexOutOfBoundsException {
```

Comment:
Why do you throw ArrayIndexOutOfBoundsException?

Reply:
this is superfluous, left when I reworked the logic of the method. Now deleted